Race/race.py

```diff
@@ -0,0 +1,75 @@
+import random
+import time
+
+
+class Animal:
+
+    def generatenumber(self):
+        num = random.randint(1, 10)
+        return num
+
+    def move(self):
+        return self.generatenumber()
+
+
+class TortoiseClass(Animal):
+    def move(self,tortoise):
+        tnum = super().move()
```

Comment:
let's use the `generatenumber()` method of the base class here. instead of `move()`